api/android/build-android-lib.sh

```diff
@@ -122,7 +123,25 @@ for arg in "$@"; do
             enable_snpe=${arg#*=}
             ;;
         --enable_tflite=*)
-            enable_tflite=${arg#*=}
+            IFS=':' read -ra enable_tflite_args <<< "${arg#*=}"
+            enable_tflite_args[2]=0
```

Comment:
Minor comment: How about just declaring extra variable instead of reusing `enable_tflite_args`? It's somewhat weird to me because this is not one of arguments.

Reply:
@dongju-chae Updated as you suggested.